evap/staff/views.py

```diff
@@ -357,6 +361,60 @@ def semester_export(request, semester_id):
 
 
 @staff_required
+def semester_raw_export(request, semester_id):
+    semester = get_object_or_404(Semester, id=semester_id)
+
+    filename = "Evaluation-%s-%s_raw.csv" % (semester.name, get_language())
+    response = HttpResponse(content_type="text/csv")
+    response["Content-Disposition"] = "attachment; filename=\"%s\"" % filename
+
+    writer = csv.writer(response, delimiter=";")
+    writer.writerow([_('Name'), _('Degrees'), _('Type'), _('Single result'), _('State'), _('#Voters'),
+        _('#Participants'), _('#Comments'), _('Average degree')])
+    for course in semester.course_set.all():
+        degrees = ",".join([degree.name for degree in course.degrees.all()])
+        course.avg_grade, course.avg_deviation = calculate_average_grades_and_deviation(course)
+        if course.state in ['evaluated', 'reviewed', 'published'] and course.avg_grade is not None:
+            course.avg_grade = ('%.1f' % float(course.avg_grade)).replace('.',',')
+        else:
+            course.avg_grade = ""
+        writer.writerow([course.name, degrees, course.type.name, course.is_single_result(), course.state,
+            course.num_voters, course.num_participants, len(course.textanswer_set.all()), course.avg_grade])
+
+    return response
+
+
+@staff_required
+def semester_participation_export(request, semester_id):
+    semester = get_object_or_404(Semester, id=semester_id)
+    courses = Course.objects.filter(semester=semester)
+    participants = set()
+    for course in courses:
+        for participant in course.participants.all():
+            if can_user_use_reward_points(participant):
+                participants.add(participant)
+    participants = sorted(participants, key=attrgetter('username'))
+
+    filename = "Evaluation-%s-%s_participation.csv" % (semester.name, get_language())
+    response = HttpResponse(content_type="text/csv")
+    response["Content-Disposition"] = "attachment; filename=\"%s\"" % filename
+
+    writer = csv.writer(response, delimiter=";")
+    writer.writerow([_('Username'), _('#Required courses voted for'), _('#Required courses'),
+        _('#Optional courses voted for'), _('#Optional courses'), _('Earned reward points')])
+    for participant in participants:
+        number_of_required_courses = Course.objects.filter(semester=semester, participants=participant, is_required_for_reward=True).count()
+        number_of_required_courses_voted_for = Course.objects.filter(semester=semester, voters=participant, is_required_for_reward=True).count()
+        number_of_optional_courses = Course.objects.filter(semester=semester, participants=participant, is_required_for_reward=False).count()
+        number_of_optional_courses_voted_for = Course.objects.filter(semester=semester, voters=participant, is_required_for_reward=False).count()
```

Comment:
`Semester.courses.filter` instead of `Course.objects.filter(semester=semester` would shorten those lines a bit